### PR TITLE
default template works on iOS 

### PR DESCRIPTION
**Pull request by drelaptop**

### CODE REVIEW

File: templates/js-template-default/frameworks/runtime-src/proj.ios_mac/ios/AppController.mm [hunk 1]
```diff
@@ -32,8 +32,6 @@ of this software and associated documentation files (the "Software"), to deal
 
 #import "cocos-analytics/CAAgent.h"
```
[the remark this answers is not in the document]
Comment (minggo) [reply templates/js-template-default/frameworks/runtime-src/proj.ios_mac/ios/AppController.mm]: Yep, remove all ANYSDK related codes.

File: templates/js-template-default/frameworks/runtime-src/proj.ios_mac/ios/AppController.mm [hunk 2]
```diff
@@ -50,15 +48,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     if (s_sharedApplication == nullptr)
     {
-        s_sharedApplication = new (std::nothrow) AppDelegate();
+        s_sharedApplication = new (std::nothrow) AppDelegate(960, 640);
```
Comment (minggo) [inline templates/js-template-default/frameworks/runtime-src/proj.ios_mac/ios/AppController.mm]: Should use the size of window.

File: templates/js-template-default/frameworks/runtime-src/proj.ios_mac/ios/RootViewController.mm [hunk 3]
```diff
@@ -84,23 +74,6 @@ - (BOOL) shouldAutorotate {
...
-    auto glview = cocos2d::Director::getInstance()->getOpenGLView();
-
-    if (glview)
-    {
-        CCEAGLView *eaglview = (__bridge CCEAGLView *)glview->getEAGLView();
-
-        if (eaglview)
-        {
-            CGSize s = CGSizeMake([eaglview getWidth], [eaglview getHeight]);
-            cocos2d::Application::getInstance()->applicationScreenSizeChanged((int) s.width, (int) s.height);
```
[the remark this answers is not in the document]
Comment (minggo) [reply templates/js-template-default/frameworks/runtime-src/proj.ios_mac/ios/RootViewController.mm]: No problem.

**Comment by drelaptop** [comment]
corrected, please review @minggo

**Pull request merged**